docs/content/oracleobjectstorage.md

```diff
@@ -151,6 +155,7 @@ Rclone supports the following OCI authentication provider.
     User Principal
     Instance Principal
     Resource Principal
+    Workload Identity
```

Comment:
to keep it consistent, can we please name it workload principal instead of workload identity everywhere. 

Workload Principal
workload_principal_auth

Reply:
As Oracle Cloud Infrastructure documentation refers to this authentication type as Workload Identity, this may be confusing to end users. See: https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contenggrantingworkloadaccesstoresources.htm

For this reason I would like to keep the terminology as "Workload Identity"

Reply:
ok that is fine then.

OKE identity is nothing but a special version of ResourcePrincipal 
It sucks that OKE team chose to name it like that in SDK: OkeWorkloadIdentityConfigurationProvider
instead of OkeWorkloadPrincipalConfigurationProvider

but in policy it is:
request.principal.type = 'workload',